Revit_Core_Engine/Query/DuctSectionProfile.cs

```diff
@@ -36,12 +36,12 @@ public static partial class Query
         /***************************************************/
         /****               Public Methods              ****/
         /***************************************************/
-
-        [Description("Section profile of round and rectangular ducts converted from a Revit duct.")]
-        [Input("revitDuct", "Revit duct to be converted into a section profile.")]
+        
+        [Description("Convert a Revit duct into a BHoM duct section profile.")]
+        [Input("revitDuct", "Revit duct to be converted into a BHoM section profile.")]
         [Input("settings", "Revit settings.")]
         [Input("refObjects", "Referenced objects.")]
-        [Output("profile", "Converted BHoM duct section profile converted.")]
+        [Output("profile", "BHoM section profile for a duct converted from a Revit duct.")]
```

Comment:
It is not conversion, rather extraction.

Reply:
Edited accordingly